src/engine/contactsdatabase.cpp

```diff
@@ -246,7 +249,7 @@ static const char *createUrlsTable =
         "\n detailId INTEGER PRIMARY KEY ASC REFERENCES Details (detailId),"
         "\n contactId INTEGER KEY,"
         "\n url TEXT,"
-        "\n subTypes TEXT);";
+        "\n subTypes TEXT);";       // Contains a (singular) INTEGER represented as TEXT (and should be named 'subType')
```

Comment:
unrelated to this but github won't let me comment on line 219, but perhaps add a comment to the plural TEXT values describing that they consist of multiple semicolon separated numbers in TEXT form.